ip_scan_topic: "scan/"
op_path_topic: "paths/"

# Ignore doors smaller than this width
min_door_width: 0.5
# Door entry/exit point distance from door
entry_path_offset: 1.0

## RANSAC line detection params
# Min number of point for a line detection
line_min_points: 20
# Min number of actual model fits for a line detection
line_exact_min_points: 15

## Eucledean clustering params
cluster_min_size: 15
cluster_max_size: 150
cluster_tolerance: 0.5
